app/views/variables/show.html.erb

```diff
@@ -13,6 +13,19 @@
         <% elsif available_provider?(@provider_id) %>
           <%= link_to 'Edit Record', '#not-current-provider-modal', class: 'display-modal not-current-provider', data: { 'provider': @provider_id, record_action: 'edit-variable' } %> |
         <% end %>
+        <% if current_provider?(@provider_id) %>
+          <%= link_to 'Delete Variable Record', "#delete-record-modal", class: 'display-modal delete-collection' %>
+        <% elsif available_provider?(@provider_id) %>
+          <%= link_to 'Delete Variable Record', '#not-current-provider-modal', class: 'display-modal not-current-provider', data: { 'provider': @provider_id, record_action: 'delete-collection' } %>
```

Comment:
would it be better to call this `delete-variable` for the modal and update `change_current_provider.coffee`, or possibly abstract it to `delete-record`?

Also, I think that the class would make the modal say 'Deleting this collection'.